Documentation/k8s-filesystem.md

```diff
@@ -105,28 +97,19 @@ spec:
         - name: REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY
           value: /var/lib/registry
         volumeMounts:
-        - name: image-store
+        - name: myfs
           mountPath: /var/lib/registry
         ports:
         - containerPort: 5000
           name: registry
           protocol: TCP
       volumes:
-      - name: image-store
-        cephfs:
-          monitors:
-          - INSERT_MONS_HERE
-          user: admin
-          secretRef:
-            name: rook-admin
-```
-
-We will need to update the yaml with the monitor IP addresses with the following commands.
-In the future this step will be improved with a Rook volume plugin.
-```bash
-cd cluster/examples/kubernetes
-export MONS=$(kubectl -n rook get service -l app=rook-ceph-mon -o json|jq ".items[].spec.clusterIP"|tr -d "\""|sed -e 's/$/:6790/'|paste -s -d, -)
-sed "s/INSERT_MONS_HERE/$MONS/g" kube-registry.yaml | kubectl create -f -
+      - name: myfs # name of the filesystem specified in the filesystem CRD.
+        flexVolume:
```

Comment:
I like how simple this gets with this PR :+1: 
No more updating the mons in the manifests!